vitest.config.ts

```diff
@@ -34,6 +23,46 @@ export default defineConfig({
         'src/utils/http-status.ts',
       ],
     },
-    pool: 'forks',
+    projects: [
+      './runtime-tests/*/vitest.config.ts',
+      {
+        esbuild: {
+          jsx: 'automatic',
+          jsxImportSource: './src/jsx',
+        },
+        extends: true,
+        test: {
+          exclude: [...configDefaults.exclude, '**/sandbox/**', '**/*.case.test.*'],
+          include: [
+            'src/**/(*.)+(spec|test).+(ts|tsx|js)',
+            'scripts/**/(*.)+(spec|test).+(ts|tsx|js)',
+            'build/**/(*.)+(spec|test).+(ts|tsx|js)',
+          ],
+          name: 'unit',
```

Comment:
I think the naming `unit` is not perfectly proper for the tests. How about `main`?